zeitschrift-fur-kunstgeschichte.csl

```diff
@@ -25,11 +25,15 @@
       <term name="note">wie Anm.</term>
       <term name="et-al">et al.</term>
       <term name="editor" form="short">Hg.</term>
+      <term name="translator" form="verb-short">übers. von</term>
+      <term name="editortranslator" form="verb-short">hg. und übers. von</term>
     </terms>
   </locale>
   <locale xml:lang="en">
     <terms>
       <term name="note">as note</term>
+      <term name="translator" form="verb-short">transl. by</term>
+      <term name="editortranslator" form="verb-short">ed. and transl. by</term>
```

Comment:
The [standard locale](https://github.com/citation-style-language/locales/blob/master/locales-en-US.xml#L311) would give:
```
<term name="translator" form="verb-short">trans. by</term>
<term name="editortranslator" form="verb-short">ed. &amp; trans. by</term>
```
Are you sure that we should choose different abbreviations here for "translated"?